pkg/e2etemplates/error.go

```diff
@@ -0,0 +1,12 @@
+package e2etemplates
+
+import "github.com/giantswarm/microerror"
+
+var invalidDataError = &microerror.Error{
+	Kind: "invalidData",
```

Comment:
That is not right IMO. The variable name should just match the kind. When the message generation hurts you so much we should rather implement the stupid hack there instead of letting this here get out of hand.

Reply:
It wasn't on purpose I just didn't setup my snippet right.

Reply:
This error has to be removed when the one below is the right one.